### ISSUE TITLE
Column names and string values are parsed incorrectly on Windows

### ISSUE THREAD

**Issue by tvandinther** [tvandinther/libsql-client-dotnet#8]
Operating system: Windows 11
Version: 0.2.2

Actual:
```
??r??? ???, ??, ?????????l??????????????'??↑??????, ??
------------------------
1, ??????, 182.6, [ a4 c7 b8 e2 ... 9a 09 3f ed ]
2, ????, 0.5, [ 00 ]
```

Expected:
```
id, name, height, data
------------------------
1, John Doe, 182.6, [ a4 c7 b8 e2 ... 9a 09 3f ed ]
2, Jane Doe, 0.5, [ 00 ]
```

**Comment by tvandinther** [comment]
I suspect it is trying to stringify a pointer. Each execution has a different series of ?????

**Comment by tvandinther** [comment]
Fixed in v0.2.3
